### ISSUE TITLE
Allow team admin to "suspend" user account, without removing them from team.

### ISSUE THREAD

**Issue by JP-from-Postman** [a85/POSTMan-Chrome-Extension#6513]
**Is your feature request related to a problem? Please describe.**
If a team member's access to all team content needs to be revoked immediately, the only current practical option is to remove them from the team entirely. However, this may not be desirable in all circumstances.

E.g. User account security is compromised, but the user was working on content in a personal workspace that will need to be transferred to team ownership before they are removed from the team.

**Describe the solution you'd like**
Add ability for a team admin (super admin?) to suspend or freeze a user account, without removing them from the team. This would block all access to team content, and perhaps freeze the account entirely (including personal workspaces and dashboard login).

**Comment by JP-from-Postman** [comment]
This may fit well with https://github.com/postmanlabs/postman-app-support/issues/5086

**Comment by sankalp0o** [comment]
@JP-from-Postman How would freezing help in recovering personal data from the compromised account?

**Comment by JP-from-Postman** [comment]
@sankalp0o Good point: this would not (on its own) enable personal content to recovered. This would really need to be paired with some ability for a team administrator to manage user accounts to be fully effective.

**Comment by JP-from-Postman** [comment]
@sankalp0o user notes from original ticket:
`I need to remove (or mark inactive) a user, blocking the user from accessing our shared collections, but I don't want his contributions or owned work to be deleted. How do I ensure this?`

`I always like being able to inactivate instead of fully remove (just in case they had owned something key we weren't aware of).`

**Comment by sankalp0o** [comment]
@JP-from-Postman thanks for the notes! This raises a couple of questions in my head:

1. A user's contributions (shared work) remains with the team even when the user leaves the team. If the team is worrying about this, it might mean one of two things - 
   a. We're not making this clear enough
   b. There is a bug which made the team lose their work 
If we know which is it, we would be able to work upon it.

2. In the current assumptions, a user's personal work is not controlled by the team. Is this an expectation from the team to be able to access a member's personal resources?

**Comment by ArjunSingh-PM** [comment]
We allow user deactivation and reactivation via [SCIM](https://learning.postman.com/docs/administration/scim-provisioning/scim-provisioning-overview/#scim-features).
